### DISCUSSION TITLE
Autoplay embedded swf?

### DISCUSSION THREAD

**Discussion by passegua** [ruffle-rs/ruffle#8096] [Q&A] [answered]
I am using the following code on my site in order to have swf running again.
It is working quite well, but I wonder if it is possible having the swf autoplay on load?
How?
Many thanks.
`<script src="https://www.sinapsi.org/public/ruffle/ruffle.js">
</script><object width="750" height="550"> <param name="movie" value="https://www.sinapsi.org/public/espressioneletterale01.swf" /> <embed src="https://www.sinapsi.org/public/espressioneletterale01.swf" /> </object>`

**Comment by n0samu** [comment]
You can use the `autoplay` configuration option, see https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configuration-options

**Reply by passegua** [reply]
Thanks n0samu, could you be so kind to write me here the modified code using that "autoplay" option? I am completely ignorant of code :-( i do not know where exactly add that setting:
`<script src="https://www.sinapsi.org/public/ruffle/ruffle.js">
</script><object width="750" height="550"> <param name="movie" value="https://www.sinapsi.org/public/espressioneletterale01.swf" /> <embed src="https://www.sinapsi.org/public/espressioneletterale01.swf" /> </object>`

**Comment by ROBERT-MCDOWELL** [comment]
@passegua 
you should add javascript code just after the ruffle.js import and before the <object>
<script>
.....
</script>

**Comment by passegua** [comment, marked as answer]
I got the final solution (working very well) from a Ruffle creator contacted through Disqus.
Just adding this piece of code the swf will autostart:

`    <script>
        window.RufflePlayer = window.RufflePlayer || {};
        window.RufflePlayer.config = {
            "warnOnUnsupportedContent": false, 
            "autoplay": "on", 
            "unmuteOverlay": "hidden"
        };
    </script>
`

**Reply by n0samu** [reply]
Discord not Disqus, and that was me who answered your question there 😉

**Reply by passegua** [reply]
Very sorry for the confusion and thanks a lot again!

**Comment by passegua** [comment]
Just added before this:
`<script src="https://www.sinapsi.org/public/ruffle/ruffle.js"></script>
<object width="750" height="550">
    <param name="movie" value="https://www.sinapsi.org/public/merlinocoord.swf">
    <embed src="https://www.sinapsi.org/public/merlinocoord.swf">
    </embed>
</object>`
